### DISCUSSION TITLE
how do I filter for a single variable from the MODIS dataset?

### DISCUSSION THREAD

**Discussion by priyapatel1** [earthaccess-dev/earthaccess#502] [Q&A]
I'm downloading data from MODIS (short name = MOD13Q1). I'm able to download ALL the data just fine. How do I only download the 16-day NDVI?

Here's a link to the dataset: https://lpdaac.usgs.gov/products/mod13q1v006/

**Comment by asteiker** [comment]
Hi @priyapatel1, I'm glad to hear that the MOD13Q1 download is working for you. It sounds like you are interested in subsetting the data by variable, in this case the 250m_16 days_NDVI layer. Currently this is not supported directly by earthaccess. [AppEEARS](https://appeears.earthdatacloud.nasa.gov/) may be a good option for you, which does provide subsetting of this collection. 

This is another good use case for #328 which would add the ability for earthaccess to leverage AppEEARS or other services to offer subsetting within your same earthaccess workflow.
